packages/language-server/src/ui5-model.ts

```diff
@@ -54,14 +56,20 @@ export async function getSemanticModelWithFetcher(
       let apiJson = await readFromCache(cacheFilePath);
       // If the file doesn't exist in the cache (or we couldn't read it), fetch it from the network
       if (apiJson === undefined) {
+        getLogger().info("No cache found for UI5 lib", { libName });
         const url = baseUrl + libName.replace(/\./g, "/") + suffix;
         const response = await fetcher(url);
         if (response.ok) {
           apiJson = await response.json();
           await writeToCache(cacheFilePath, apiJson);
         } else {
-          console.error(`Could not read UI5 resources from ${url}`);
+          getLogger().error("Could not read UI5 resources from", { url });
```

Comment:
Currently with the noop logger, we won't write anything anywhere on these errors. Maybe instead of noop it should at least print to the console? Until we have the implementation that prints to a file (and maybe also the console so the user can easily see such errors)

Reply:
print to the console will be included in this PR (hopefully tonight)